src/cmd/devdraw/wayland.c

```diff
@@ -422,27 +427,59 @@ static const struct xdg_toplevel_listener xdg_toplevel_listener = {
 	.close = xdg_toplevel_close,
 };
 
-static const struct wl_callback_listener wl_callback_listener;
+static const struct wl_callback_listener wl_callback_key_repeat_listener;
+
+static void wl_callback_key_repeat(void *data, struct wl_callback *wl_callback, uint32_t time) {
+	int dt = 0;
```

Comment:
I think in the rest of the file here I've been declaring variables at their first use instead of the older style of declaring everything at the top of the block like this. Mind moving these down to their use?

Reply:
I moved them closer to their first use. The repetitions and repeat_rune have to be declared before checking that repeat is active so they aren't used uninitialized.